.github/workflows/lib-tests.yaml

```diff
@@ -58,6 +58,9 @@ jobs:
         env:
           test_dir: /github/workspace/libs/${{ matrix.library }}
         run: |
+            apt-get update
```

Comment:
what's the reason for needing this install?

Reply:
pycbc depends on ligo-segments, which required gcc to be installed for whatever reason

Reply:
Got it, does this break our local poetry installs of things that depend on these libs?

Reply:
No, those still work fine, though I guess it's something to look out for